app/views/users/show.html.erb

```diff
@@ -71,7 +71,7 @@
         row.key { 'Key contact' }
         row.value { @user.is_key_contact? ? "Yes" : "No" }
         if current_user.data_coordinator?
-          row.action(visually_hidden_text: "are #{current_user == @user ? "you" : "they"} a key contact?", href: edit_user_path, html_attributes: { "data-qa": "change-are-#{current_user == @user ? "you" : "they"}-a-key-contact" })
+          row.action(visually_hidden_text: "are #{pronoun(@user, current_user)} a key contact?", href: aliased_user_edit(@user, current_user), html_attributes: { "data-qa": "change-are-#{pronoun(@user, current_user)}-a-key-contact" })
         else
           row.action()
         end
```

Comment:
It feel like there's a lot of logic living in this view file

Reply:
I wonder if any of it could be extracted to the user helper? Just pondering here but not a strong opinion

Reply:
Tried extracting out the row action creation but I think that gets messier because then you're passing the row element into the helper method which leads to lots of mocking/stubbing for testing. Or you create the whole summary list row in the helper but then you're maybe just moving all the logic from one file to another? Didn't find a great answer...